membership-common/src/main/scala/com/gu/memsub/Plan.scala

```diff
@@ -3,18 +3,15 @@ package com.gu.memsub
 sealed trait Status {
   def name: String
 }
-case class Legacy() extends Status {
```

Comment:
I guess someone didn't know how to do Current.type to get the type of a case object, hence it was done this way.